package.json

```diff
@@ -42,12 +42,12 @@
   ],
   "license": "Apache-2.0",
   "devDependencies": {
-    "@airbnb/config-babel": "^2.1.3",
-    "@airbnb/config-eslint": "^2.1.3",
-    "@airbnb/config-jest": "^2.1.3",
-    "@airbnb/config-prettier": "^2.0.4",
-    "@airbnb/config-typescript": "^2.1.2",
-    "@airbnb/nimbus": "^2.1.3",
+    "@airbnb/config-babel": "2.2.4",
```

Comment:
Why pin to specific version without `^`?

Reply:
Was having trouble building. Couldn't figure out why. Just don't want to deal with the unpredictability. Sometimes even with the implied "not a breaking change" in a minor version upgrade, things might still break.

Reply:
This repo already check in `yarn.lock` so if you did not modify these package version, it should be the same version that was working in `master`. Could you try reseting `yarn.lock` and `package.json` to `master`'s version while keeping the changes you made to `BigNumber`'s `package.json` and `yarn install` again?

Reply:
Weird, it works... I'm pretty sure I tried that yesterday... Maybe it's because I inadvertently pined `@types/shortid`.  

I think it's probably not a bad idea to pin `devDependencies` as they are unlikely to cause dependency hell and the added stability is valuable. Renovate actually [pin the dependencies for you](https://github.com/ktmud/incubator-superset/pull/2) once you set it up.